src/components/CustomerQuota/CheckoutSuccess/CheckoutSuccessCard.tsx

```diff
@@ -170,20 +167,14 @@ export const CheckoutSuccessCard: FunctionComponent<CheckoutSuccessCard> = ({
   }, [error, showErrorAlert]);
 
   const translationProps = useTranslate();
+  const { c13nt, i18nt } = translationProps;
```

Comment:
for line 270, maybe can refactor to `i18nt` instead of `translationProps.i18nt` ?